activation/activation.go

```diff
@@ -39,14 +39,22 @@ var (
 
 // PoetConfig is the configuration to interact with the poet server.
 type PoetConfig struct {
-	PhaseShift                     time.Duration `mapstructure:"phase-shift"`
-	CycleGap                       time.Duration `mapstructure:"cycle-gap"`
-	GracePeriod                    time.Duration `mapstructure:"grace-period"`
-	RequestTimeout                 time.Duration `mapstructure:"poet-request-timeout"`
-	RequestRetryDelay              time.Duration `mapstructure:"retry-delay"`
+	DefaultRequestTimeout time.Duration `mapstructure:"poet-client-default-request-timeout"`
+	RequestRetryDelay     time.Duration `mapstructure:"poet-client-retry-delay"`
```

Comment:
We have different kind of poet config parameters:
- general parameters for http client
- parameters for registration 

By reading just the config it wasn't obvious 'who is who' and sometimes naming is confusing. 
I can revert naming for already existing variables, to avoid unnecessary changes. 
But I would put some task in backlog to think about namings refactoring to make it more obvious and matching with naming of entities in out documentations.

Reply:
Updating the `CHANGELOG` will be needed anyway. Sure there is a technical difference between some of the parameters, but from the users/operators point of view I struggle to see this. Maybe some like PhaseShift and CycleGap are defined by the poet and must not be changed by the user and some like timeouts are free to be adjusted.

But again changing any of those names must be documented and added to `CHANGELOG` and mentioned in the release notes. Existing nodes will not start any more after upgrading if they don't also update their configuration and users need to be informed about this.